### PR TITLE
Create PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by luisbevilacqua**
## Objective
Create a template for Pull Requests to make contributing to the project easier.

## Notes
The **Notes**, **How to test** and **Breaking changes** sections are meant to be optional but are strongly encouraged.
This file is inside the `.github/` folder to avoid cluttering the root directory, following the Github documentation: https://github.com/blog/2111-issue-and-pull-request-templates

## Checklist

- [x] I have read the **[CONTRIBUTING]** document.
- [x] I have updated the documentation accordingly. (If needed)
- [ ] I have created a _Codelog changefile_ with the changes made to the branch.
- [ ] I have created a test proving that my feature/fix does what it intends to do.

[CONTRIBUTING]: https://github.com/codus/codelog/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+<!--- Provide an unambiguous set of steps to correctly setup and test the changes: -->
+1.
+2.
+3.
+4.
+
+## Breaking changes
+<!--- Does these changes make updating from older versions problematic? -->
+<!--- Is any data lost if updated? -->
+<!--- If so what additional steps should users take to update? -->
```
Comment (emilianoLeite) [inline .github/PULL_REQUEST_TEMPLATE.md]: @luisbevilacqua 
I would change this line to
> If so **,** what additional steps should users take **when updating**?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+## Checklist
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+- [ ] I have read the **[CONTRIBUTING]** document.
+- [ ] I have created a _Codelog changefile_ with the changes made to the branch.
+- [ ] I have created a test proving that my feature/fix does what it intends to do.
+- [ ] I have updated the documentation accordingly. (If needed)
+
+[CONTRIBUTING]: https://github.com/codus/codelog/blob/master/CONTRIBUTING.md
```
Comment (emilianoLeite) [inline .github/PULL_REQUEST_TEMPLATE.md]: @luisbevilacqua     
What about moving this link to the checklist item in line 26?

**Comment by emilianoLeite** [comment]
👍

**Comment by FernandaThomazinho** [comment]
@luisbevilacqua should we add to the checklist items like Test suite is passing and No rubocop offences or is enough that it is mentioned on Contributing?

**Comment by luisbevilacqua** [comment]
@FernandaThomazinho the checklist was meant as a way to enforce certain patterns and to help the reviewers to have a visual reference of what might be missing. As we already have TrevisCI running the automated tests and linting, I don`t see the need to also add this to the checklist, as any PR that don`t fulfill those requirements are automatically blocked to merge by Github.

**Pull request merged**